sklearn/multioutput.py

```diff
@@ -0,0 +1,214 @@
+"""
+MultiOneVsRestClassifier
+===========================
+This module includes several classes that extend base estimators to
+multi-target estimators.
+
+Most sklearn estimators use a response matrix to train a target function
+with a single output variable.  I.e. typical estimators use the
+training set X to estimate a target function f(X) that
+predicts a single Y.  The purpose of this class is to extend estimators
+to be able to estimate a series of target functions (f1,f2,f3...,fn)
+that are trained on a single X predictor matrix to predict a series
+of reponses (y1,y2,y3...,yn).
+"""
+
+# Author: Hugo Bowne-Anderson <hugobowne@gmail.com>
+# Author: Chris Rivera <chris.richard.rivera@gmail.com>
+# Author: Michael Williamson
+# Author: James Ashton Nichols <james.ashton.nichols@gmail.com>
```

Comment:
I don't think that I have added anything on my own as of now. Please do it only when you people want to give a :+1 :)